provider/README.md

```diff
@@ -120,17 +120,42 @@ Data: `{ "trips": [] }`, an array of objects with the following structure
 
 ### Routes
 
+<<<<<<< HEAD
 To represent a route, MDS `provider` APIs should create a GeoJSON Feature Collection, which includes every observed point in the route, and a timestamp.
+||||||| merged common ancestors
+To represent a route, MDS `provider` APIs should create a GeoJSON Feature Collection, which includes every observed point in the route, and a timestamp. 
+=======
+To represent a route, MDS `provider` APIs should create a GeoJSON Feature Collection, which includes every observed point in the route. Each point is expressed as a GeoJSON `Feature` with a `Point` geometry.
+
+Route point features must include at least a UTC timestamp and accuracy radius in the `properties` key. Optional fields may be omitted or use the `null` value.
+>>>>>>> Add point accuracy & define optional point feature properties
```

Comment:
It looks like you rebased, but didn't resolve the merge conflicts?

Reply:
Ha! Looks like I committed before I saved the resolution. Pushed again.